keps/sig-scheduling/3101-pod-to-node-assignment/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: 1:1 Pod to Node Assignment
+kep-number: 3101
+authors:
+  - "@ahmad-diaa"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-node
```

Comment:
@ehashman @SergeyKanzhelev can one please review and ack this proposal? the kubelet change here is that the new filter will be added to the set of scheduler predicates that kubelet imports and runs on admission.

Reply:
All questions I have are for the scenario and scheduling gotchas that may be introduced by this. From kubelet implementation perspective I don't see major concerns